### ISSUE TITLE
Dryad Module

### ISSUE THREAD

**Issue by timcdlucas** [zoonproject/zoon#317]
Read in data using a DOI

**Comment by AugustT** [comment]
Nope. Data are in non-standard format. See rdryad for ROpenSci package https://github.com/ropensci/rdryad
